### PR TITLE
add io.github.lemmygtk.lemoa

### PR DESCRIPTION
**Pull request by Bnyro**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

File: io.github.lemmygtk.lemoa.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        "org.freedesktop.Sdk.Extension.llvm14"
+    ],
+    "command": "lemoa",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--device=dri",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--env=RUST_LOG=lemoa=debug",
```
Comment (hfiguiere) [inline io.github.lemmygtk.lemoa.json:17]: debug log when running the flatpak, I don't think this is usefull nor correct.

File: io.github.lemmygtk.lemoa.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            "name": "lemoa",
+            "buildsystem": "meson",
+            "builddir": true,
+            "config-opts": [
+                "-Dprofile=default"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "path": "https://github.com/lemmygtk/lemoa.git",
```
Comment (hfiguiere) [inline io.github.lemmygtk.lemoa.json:42]: this build will no work as it is done offline.

File: io.github.lemmygtk.lemoa.json (hunk 1 above)
Comment (Bnyro) [reply io.github.lemmygtk.lemoa.json]: You're right, my bad.

File: io.github.lemmygtk.lemoa.json (hunk 2 above)
Comment (hfiguiere) [reply io.github.lemmygtk.lemoa.json:41]: you need to either generate a source list with the `flatpak-cargo-generator` https://github.com/flatpak/flatpak-builder-tools/tree/master/cargo or use a tarball with vendored crates.

Comment (Bnyro) [reply io.github.lemmygtk.lemoa.json:41]: So we should add
```
"build-args": [
            "--share=network"
        ],
```
?

Comment (hfiguiere) [reply io.github.lemmygtk.lemoa.json:41]: no. this is not allowed either. 

I mentionned the two possible solutions.

Comment (Bnyro) [reply io.github.lemmygtk.lemoa.json:41]: Sorry, I didn't yet see your message when answering

Comment (Bnyro) [reply io.github.lemmygtk.lemoa.json:41]: Should I open an issue at the `flatpak-builder-tools` repo?

**Comment by Bnyro** [comment]
bot, build io.github.lemmygtk.lemoa

**Comment by hfiguiere** [comment]
```
Installing /run/build/lemoa/_flatpak_build/data/io.github.lemmygtk.lemoa.desktop to /app/share/metainfo
```

this is the wrong destination....

**Comment by Bnyro** [comment]
bot, build io.github.lemmygtk.lemoa

**Pull request closed without merging**